paperqa/prompts.py

```diff
@@ -38,14 +38,19 @@
     "If the context provides insufficient information reply "
     f'"{CANNOT_ANSWER_PHRASE}." '
     "For each part of your answer, indicate which sources most support "
-    "it via citation keys at the end of sentences, "
-    "like {example_citation}. Only cite from the context "
-    "below and only use the valid keys. Write in the style of a "
-    "Wikipedia article, with concise sentences and coherent paragraphs. "
-    "The context comes from a variety of sources and is only a summary, "
-    "so there may inaccuracies or ambiguities. If quotes are present and "
-    "relevant, use them in the answer. This answer will go directly onto "
-    "Wikipedia, so do not add any extraneous information.\n\n"
+    "it via citation keys at the end of sentences, like {example_citation}. "
+    "You can only use this formatting for citations. "
```

Comment:
Can you say `"You can only use this formatting for citations, do not apply it to X or Y."` (and add what X or Y are)